api/health.yaml

```diff
@@ -0,0 +1,38 @@
+openapi: 3.0.0
```

Comment:
This is also an api documentation, kindly update the api.yaml file instead

Reply:
done, will update soon